### PR TITLE
switch to official xgboost package

### LINKED ISSUE #143: Use official xgboost released jar
Move away from `xgboost-dist` in favor of the now official released package.

We will need to probably package `OpenMP` lib together with `zoltar-xgboost`.

### PR DESCRIPTION
**Pull request by regadas**
Fixes #143.

linux libgomp1 binary is packaged together with `zoltar-xgboost`.

### CODE REVIEW

File: scripts/libgomp/create.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+set -o nounset
+set -o pipefail
+
+DIR=$(dirname "$0")
+OUTPUT_PATH=$DIR/../../zoltar-xgboost/src/main/resources/lib/libgomp.so.1
+IMAGE_NAME=debian-libgomp1
+
+docker build -t $IMAGE_NAME $DIR
+
+docker cp -L $(docker create $IMAGE_NAME):/lib/x86_64-linux-gnu/libgomp.so.1 $OUTPUT_PATH
```
Comment (ravwojdyla) [inline scripts/libgomp/create.sh]: Do we need mac version?

Comment (regadas) [reply scripts/libgomp/create.sh]: @ravwojdyla according to the docs `OpenMP` seems to be not enabled by default.

https://xgboost.readthedocs.io/en/latest/jvm/index.html#enabling-openmp-for-mac-os

Comment (regadas) [reply scripts/libgomp/create.sh]: mentioned file https://github.com/dmlc/xgboost/blob/master/jvm-packages/create_jni.py#L73

Comment (regadas) [reply scripts/libgomp/create.sh]: given this not sure if we should worry about this for now.

Comment (ravwojdyla) [reply scripts/libgomp/create.sh]: @regadas dope! have you tested it?

Comment (regadas) [reply scripts/libgomp/create.sh]: @ravwojdyla yup, tests also confirm it, `OpenMP` is loaded by default for `xgboost` on linux.

https://github.com/spotify/zoltar/blob/cbfc4be173270f731db9ae2029291a01d30a3aa7/zoltar-xgboost/src/main/java/com/spotify/zoltar/xgboost/XGBoostModel.java#L59-L67

**Pull request merged**